lib/videojs-playlist-ui.js

```diff
@@ -67,7 +67,7 @@ const createThumbnail = function(thumbnail) {
 };
 
 videojs.PlaylistMenuItem = videojs.Component.extend({
-  init: function(player, options) {
+  init: function(player, options, settings) {
```

Comment:
The `options` argument here is the playlist item object containing `description`, `name` and `duration`, where as `settings` argument here is the "options" to the plugin - `playOnSelect`, `className`. Should we rename `options` argument to `playlistItem` ?